spec/report_spec.rb

```diff
@@ -58,6 +58,32 @@
       expect(report).to be_an_instance_of(described_class)
       expect(report.id).to match('trkrep')
     end
+
+    it 'creates a report with custom columns' do
+      report = described_class.create(
+        start_date: Fixture.report_start_date,
+        end_date: Fixture.report_end_date,
+        type: 'shipment',
+        columns: ['usps_zone'],
+      )
+
+      expect(report).to be_an_instance_of(described_class)
+      # verify params by checking URL in cassette
+      # can't do any more verification without downloading CSV
```

Comment:
Nit: Can we move these above the assertion?

Reply:
Resolved in 865b4ec1671f3fadcf24c89b6a5644a9c898a4ed